rules/sqlperf-reduce-table-size/rule.md

```diff
@@ -0,0 +1,36 @@
+---
+seoDescription: Do you keep your database tables small?
+type: rule
+archivedreason:
+title: Do you keep your database tables small?
```

Comment:
@brydeno 

Don't think we need the word "your" in this title

Reply:
Remnoved.